pandas/core/series.py

```diff
@@ -197,8 +197,12 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
             elif isinstance(data, SingleBlockManager):
                 if index is None:
                     index = data.index
-                else:
-                    data = data.reindex(index, copy=copy)
+                elif not data.index.equals(index) or copy:
+                    # GH#19275 SingleBlockManager input should only be called
+                    # internally
```

Comment:
If anything this would be an AssertionError. But is this code path actually hit by *any* tests?

Reply:
Nope